src/openrpc/chains/linea/linea.yaml

```diff
@@ -34,7 +34,6 @@ methods:
   - $ref: ../_components/evm/methods.yaml#/components/methods/eth_call
   - $ref: ../_components/evm/methods.yaml#/components/methods/eth_callMany
   - $ref: ../_components/evm/methods.yaml#/components/methods/eth_chainId
-  - $ref: ../_components/linea/methods.yaml#/components/methods/eth_estimateGas
```

Comment:
Hmm, it should have left this method since its in the ignore array. I'll investigate.